docs/documentation_dev/07_i18n.md

```diff
@@ -1,10 +1,14 @@
+---
+next: false
```

Comment:
what does this mean?

Reply:
You can use `previous` and `next` to define or turn off Previous/Next links on the page (see https://vuepress.vuejs.org/theme/default-theme-config.html#prev-next-links). Since i18n is the last entry for AsTeRICS Grid, the "next" link should not be displayed within this page.

Reply:
ok, thanks.